dist/coins-logon-widget.css

```diff
@@ -335,25 +323,21 @@ input[type="text"].coins-logon-widget-input {
  * button display horizontally.
  */
 .coins-logon-widget-form-horizontal {
-  -webkit-align-items: flex-start;
-      -ms-flex-align: start;
-          align-items: flex-start;
-  display: -webkit-flex;
+  -ms-flex-align: start;
```

Comment:
Why did you remove the webkit stuff? Shouldn't we keep it for cross browser support?

Reply:
It wasn't intentional. I just committed the dist output generated from the `npm run build`